File: Sharp/SHARP/ViewModels/Organization/OrganizationDetailedModel.cs
using System.Collections.Generic;

namespace SHARP.ViewModels.Organization
{
    public class OrganizationDetailedModel
    {
        public int Id { get; set; }

        public string Name { get; set; }
        public string OperatorName { get; set; }
        public string OperatorEmail { get; set; }

        public bool AttachPortalReport { get; set; }
        public int FacilityCount { get; set; }

        public IEnumerable<RecipientModel> Recipients { get; set; }

        public IEnumerable<OrganizationPortalFeatureModel> PortalFeatures { get; set; }

    }
}

File: AngularJSAspNetcoreSample/SHARP/ClientApp/src/app/models/organizations/organization.detailed.model.ts
import { RecipientModel } from "./recipient.model";

export interface PortalFeature {
  id: number;
  name: string;
  available: boolean;
}
export interface OrganizationDetailed {
  id: number;
  name: string;
  operatorEmail: string;
  operatorName: string;
  attachPortalReport: boolean;
  facilityCount: number;
  recipients: RecipientModel[];
  portalFeatures: PortalFeature[];
}
